iceberg/openhouse/internalcatalog/src/test/java/com/linkedin/openhouse/internal/catalog/model/HouseTableTest.java

```diff
@@ -33,9 +32,7 @@ public void testHouseTableDefaultValues() {
             continue;
           }
 
-          if (fieldName.equals("storageType")) {
```

Comment:
The right way would be to make the mapper take fileIO as an input and during the mapping, use that fileIO to get the right type

Reply:
Updated. PTAL.